notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    # Create quantization config (default) and OVQuantizer
</pre><p>could you please provide some details about configuration and how it can be used (.e.g. which parameters can be additionally set?), for some users can be unclear why it is required create instance of class without any parameters and why it can not be done automatically inside pipeline)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/774/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I've added extra info about the config. It is not possible to explain all the possible configurations here. The default should work OOB and, btw, I can remove it at all so that it will be created inside the quantize method.